dev-utils/docker-compose.yml

```diff
@@ -125,10 +125,6 @@ services:
           - wait-for-apm-server
 
   node-puppeteer:
-    build:
-      context: ../.ci/docker/node-puppeteer
-      args:
-        - NODEJS_VERSION=12
```

Comment:
will this avoid the building of those docker images?

Reply:
That's correct, it was not only the script but the compose file. Submitting a fix now